src/components/Navigation/DesktopNavigation/index.js

```diff
@@ -43,12 +114,14 @@ DesktopNavigation.propTypes = {
   logoProps: PropTypes.shape({}),
   menuProps: PropTypes.arrayOf(PropTypes.shape({})),
   socialLinks: PropTypes.arrayOf(PropTypes.shape({})),
+  isExplore: PropTypes.bool,
```

Comment:
Rather than `isExplore` boolean, I think `variant` would be better... or even a separate Navigation component altogether since they only share logo with the current navigation... and logo is already a reusable component.